src/Vault.sol

```diff
@@ -522,18 +551,27 @@ contract Vault is IERC4626, ERC20Permit, ILiquidationSource, IClaimable, Ownable
     uint8 _v,
     bytes32 _r,
     bytes32 _s
-  ) external onlyVaultCollateralized returns (uint256) {
+  ) external returns (uint256) {
+    uint256 _depositedAssets = _totalSupply();
+    uint256 _withdrawableAssets = _totalAssets();
+
     _permit(IERC20Permit(address(_asset)), _owner, address(this), _assets, _deadline, _v, _r, _s);
-    return _depositAssets(_assets, _owner, _owner);
+
+    return _depositAssets(_assets, _owner, _owner, _depositedAssets, _withdrawableAssets);
```

Comment:
can inline

Reply:
Fixed in this commit: https://github.com/GenerationSoftware/pt-v5-vault/pull/51/commits/029a830aec2a6519a7c7375dec0ccde99d0e9ef5